src/scripts/main.js

```diff
@@ -1,3 +1,248 @@
 'use strict';
 
-// write your code here
+const fieldRows = document.querySelectorAll('.field_row');
+const messageWin = document.querySelector('.message_win');
+const messageLose = document.querySelector('.message_lose');
+const button = document.querySelector('.start');
+const messageStart = document.querySelector('.message_start');
+const gameScore = document.querySelector('.game_score');
+
+const GRID_SIZE = 4;
+
+document.addEventListener('keydown', (e) => {
+  if (e.key === 'ArrowUp') {
+    grid.slideUp();
+  } else if (e.key === 'ArrowLeft') {
```

Comment:
Use switch case construction. Move all cases to the object.

Example of usage:

`case Directions.ArrowLeft:`